.github/workflows/pre_release.yaml

```diff
@@ -0,0 +1,135 @@
+name: Create a pre-release
+
+on:
+    # Push to master will deploy a beta version
+    push:
+        branches:
+            -   master
+        tags-ignore:
+            -   "**" # Ignore all tags to prevent duplicate builds when tags are pushed.
+
+concurrency:
+    group: release
```

Comment:
```suggestion
    group: prerelease
```

Reply:
Aaakchuallyy... both of those workflows add a commit to `master`, so they should be mutually excluded. I'm open to a better name for both though.

Reply:
Mmm, that is rough. Lets leave it as is but I'm worried that some day it'll try to release and prerelease at the same time and break smth

Reply:
If that happens, one of your builds may crash. I believe it won't cause anything brutal.